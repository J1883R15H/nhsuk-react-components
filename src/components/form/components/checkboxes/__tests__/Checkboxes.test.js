import React from 'react';
import { shallow } from 'enzyme';
import Checkboxes from '..';

describe('Checkboxes', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Checkboxes></Checkboxes>);
    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });
});
