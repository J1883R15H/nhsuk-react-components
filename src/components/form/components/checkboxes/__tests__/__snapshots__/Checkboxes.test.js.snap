// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkboxes matches snapshot 1`] = `
<div
  className="nhsuk-checkboxes"
  onChange={[Function]}
>
  <ContextProvider
    value={
      Object {
        "onChange": [Function],
      }
    }
  />
</div>
`;
